10_nwis_pull/src/nwis_combine_functions.R

```diff
@@ -4,22 +4,22 @@ combine_nwis_data <- function(ind_file, ...){
   
   rds_files <- c(...)
   df_list <- list()
-  
+  message('reading in partition files...')
   for (i in seq_len(length(rds_files))){
-    
+    print(rds_files[i])
     flow_dat <- readRDS(rds_files[i]) 
     
     reduced_dat <- convert_to_long(flow_dat)
     
     df_list[[i]] <- reduced_dat
   }
   
-  nwis_df <- do.call("bind_rows", df_list)
-  
+  nwis_df <- data.table::rbindlist(df_list)
```

Comment:
You don't actually need do.call with `bind_rows`. You can just write as `bind_rows(df_list)`. Which is something I only recently cemented into my brain (clearly wasn't there when we wrote this last summer). 

Is `rbindlist(df_list)` faster than `bind_rows(df_list)`?

Reply:
Yeah, it is quite a bit faster in general.   I didn't compare specifically here though.

Reply:
I will need to keep `rbindlist` in mind for the future :)